paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetLoader.kt

```diff
@@ -71,6 +76,11 @@ internal class DefaultCustomerSheetLoader(
                 configuration = configuration,
                 elementsSessionWithMetadata = elementsSessionWithMetadata,
             ).getOrThrow()
```

Comment:
Maybe `elements.customer_sheet.loading.payment_methods_load_failure`?

Reply:
Should note that after `CustomerSession` is merged, merchants using it will be loading payment methods from elements sessions exclusively so these two operations would be merged.